### PR TITLE
fix(subcomponents): Make all subcomponents available from base component

### PR DESCRIPTION
**Pull request by jeff-phillips-18**
This allows all subcomponents to be available from the pf-react packages.

@mareklibra

### CODE REVIEW

**Comment by jeff-phillips-18** [comment]
See https://github.com/patternfly/patternfly-react/pull/894#discussion_r232415398 for context.

File: packages/patternfly-3/patternfly-react/src/components/TypeAheadSelect/TypeAheadSelect.js [hunk 1]
```diff
@@ -19,4 +20,6 @@ TypeAheadSelect.asyncContainer = asyncContainer;
 TypeAheadSelect.menuItemContainer = menuItemContainer;
 TypeAheadSelect.tokenContainer = tokenContainer;
 
+TypeAheadSelect.Async = AsyncTypeAheadSelect;
+
```
Comment (cdcabrera) [inline packages/patternfly-3/patternfly-react/src/components/TypeAheadSelect/TypeAheadSelect.js]: Note: Unsure if we want the `GithubMenuItem` exposed...

File: packages/patternfly-3/patternfly-react/src/components/Cards/Card.js [hunk 2]
```diff
@@ -63,7 +66,12 @@ Card.Body = CardBody;
 Card.Heading = CardHeading;
 Card.Footer = CardFooter;
 Card.Link = CardLink;
+Card.Grid = CardGrid;
 Card.DropdownButton = CardDropdownButton;
 Card.HeightMatching = CardHeightMatching;
+Card.UtilizationTrend = UtilizationTrendCard;
```
[the remark this answers is not in the document]
Comment (jeff-phillips-18) [reply packages/patternfly-3/patternfly-react/src/components/Cards/Card.js]: The thought here is that that they are available via the UtilizationCard already, but I guess it makes sense to make them available individually as well.

File: packages/patternfly-3/patternfly-react/src/components/Label/Label.js [hunk 3]
```diff
@@ -20,10 +23,15 @@ Label.propTypes = {
...
 Label.defaultProps = {
   children: null,
   type: 'default',
   onRemoveClick: undefined
 };
 
+Label.RemoveButton = RemoveButton;
+Label.DisposableLabel = DisposableLabel;
+Label.CompoundLabel = CompoundLabel;
+
```
[the remark this answers is not in the document]
Comment (jeff-phillips-18) [reply packages/patternfly-3/patternfly-react/src/components/Label/Label.js]: Nice catch, missed that one.

File: packages/patternfly-3/patternfly-react/src/components/TypeAheadSelect/TypeAheadSelect.js (hunk 1 above)
Comment (jeff-phillips-18) [reply packages/patternfly-3/patternfly-react/src/components/TypeAheadSelect/TypeAheadSelect.js]: I don't think so, its only used in the storybook.

**Pull request merged**